index.html

```diff
@@ -13,48 +14,112 @@
+    </section>
+    <section class="review">
+      <h2 class="witheader">How Do You Sleep At Night? - Review</h2>
+      <img src="./images/hdysan.png" height="400" alt="Album-Cover" class="witimg"/>
+      <p class="wittext">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis lectus lorem, mattis vel eros sagittis, mattis accumsan dui. Sed vel dui nulla. Curabitur at pellentesque turpis, at auctor massa. Nullam in quam at enim aliquet accumsan a sit amet est. Vivamus enim mi, feugiat eu interdum ut, gravida nec dolor. In egestas magna odio. Mauris tincidunt nec mauris a ultricies. Donec urna lacus, tincidunt a mauris vitae, finibus laoreet magna. Sed ac ipsum sit amet ex maximus ullamcorper. Morbi vehicula et dui non efficitur. In mauris lectus, blandit quis arcu et, venenatis pretium magna.</p>
+    </section>
+    <section class="rating">
+      <h2 class="wittext">Final Rating</h2>
+      <img src="./images/rating.png" height="100" alt="4.5-stars" class="witimg"/>
+      <p class="wittext">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis lectus lorem, mattis vel eros sagittis, mattis accumsan dui. Sed vel dui nulla. Curabitur at pellentesque turpis, at auctor massa.</p>
+    </section>
+    <section class="form">
+      <H2>Enter Our Free Vinyl Giveaway!</H2>
+      <form>
+        <div>
+        <label>Please enter your email</label>
+        <input type="text" placeholder="johndoe@example.com" autocomplete="email">
+        </div>
+        <br>
+        <div>
+          <label>Please enter your phone number</label>
+          <input type="text" placeholder="(1)-888-888-8888" autocomplete="tel">
+        </div>
+        <br>
+        <h3>Select Vinyl(s)</h3>
+        <div>
+          <label>Travis Scott's Utopia</label>
+          <input type="checkbox">
+        </div>
+        <br>
+        <div>
+          <label>Dominic Fike's Sunburn</label>
+          <input type="checkbox">
+        </div>
+        <br>
+        <div>
+          <label>Teezo Touchdown's How Do You Sleep at Night</label>
+          <input type="checkbox">
+        </div>
+        <br>
+        <div>
+          <input type="submit" value="Submit">
+        </div>
+      </form>
+    </section>
+  </main>
+  <footer>
+    <section>
+      <nav>
+        <h4>Site navigation</h4>
+        <ul>
```

Comment:
Missing an `h2` to start your footer section 

Also you have to follow logical heading order - so you can't just show up with an h4 without having an h2..h3..h4...